data/modules/alpha.yml

```diff
@@ -45,21 +45,44 @@ alpha:
     type: sub_module_intro
   1-2-1:
     type: text_page # topic?
-  1-2-2:
+  1-2-1-1:
     type: formative_assessment
-  1-2-3:
+  1-2-1-2:
     type: youtube_page
-  1-2-4:
+  1-2-1-3:
     type: formative_assessment
+
+  # ----------------------------------------------------------------------------
+  # Submodule 3 summative assessment
+  # ----------------------------------------------------------------------------
+
+  1-3:
+    type: sub_module_intro
   1-3-1:
-    type: text_page
+    type: sub_module_intro
   1-3-2:
+    type: sub_module_intro
+  1-3-2-1:
     type: summative_assessment
-  1-3-3:
+  1-3-2-2:
     type: summative_assessment
-  1-3-4:
+  1-3-2-3:
     type: summative_assessment
-  1-3-5:
+  1-3-2-5:
     type: assessments_results
-  1-3-6:
-    type: text_page
\ No newline at end of file
+
+
+  # ----------------------------------------------------------------------------
+  # Submodule 4 Confidence check end summary
+  # ----------------------------------------------------------------------------
+
+  1-3-3:
+    type: sub_module_intro
+  2-3-3-1:
```

Comment:
How come these start with a 2 rather than a 1?

Reply:
You are right should start with 1 updated now.